lib/src/migrator.dart

```diff
@@ -55,6 +63,16 @@ abstract class Migrator extends Command<Map<Uri, String>> {
         allMigrated[file] = migrated[file];
       }
     }
+    if (missingDependencies.isNotEmpty) {
+      var count = missingDependencies.length;
+      emitWarning(
+          "$count dependenc${count == 1 ? 'y' : 'ies'} could not be found.");
+      if (globalResults['verbose'] as bool) {
+        for (var warning in missingDependencies) {
+          print('  $warning');
+        }
+      }
+    }
```

Comment:
Also consider abstracting this condition into a separate method to make this method shorter.